client/src/components/CKANRegistryDisplay.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
I think it'd be a good idea to separate the frontend components from the CMS components. In #121 I've done the same thing, and put the frontend components in `client/src/Frontend/` for example

Reply:
I wondered the same, but after a bit of open room consulting, decided not to.
It's worth having the wider discussion though.

The general consensus was:
* Components are components. It does not particularly matter where they are used.
* Managing the separation can get tricky
* Configuration may get overly complicated.